src/audio/sfx.rs

```diff
@@ -54,23 +54,23 @@ impl SoundEffectHandles {
 }
 
 /// A custom command used to play sound effects.
-struct PlaySoundEffect {
+struct PlaySfx {
     name: String,
     settings: PlaybackSettings,
 }
 
-impl Command for PlaySoundEffect {
+impl Command for PlaySfx {
     fn apply(self, world: &mut World) {
         // If you need more complex behavior, use `world.run_system_once_with`,
         // as demonstrated with `PlaySoundtrack`.
-        world.resource_scope(|world, mut sound_effects: Mut<SoundEffectHandles>| {
+        world.resource_scope(|world, mut sound_effects: Mut<SfxHandles>| {
             sound_effects.play(self.name, world, self.settings);
         });
     }
 }
 
 /// An extension trait with convenience methods for sound effect commands.
-pub trait SoundEffectCommands {
+pub trait SfxCommands {
     fn play_sound_effect_with_settings(
```

Comment:
missed (and below)

Reply:
[`3d1ecd6` (#244)](https://github.com/TheBevyFlock/bevy_quickstart/pull/244/commits/3d1ecd65af454807a834168e5b07c47a0f11d92b)